README.md

```diff
@@ -27,6 +28,22 @@ This API is being used by 3 frontend projects currently being developed:
 - [Contact](#contact)
 - [License](#license)
 
+## Features
+
+- Login and register a new user
+- Create a user profile, view and edit user profiles
+- Change user password, refresh jwt tokens
+- Create relation between two users with a fixed period of time
+- Both mentor and mentee in a relation can create tasks
+- Assign and remove admin roles to users and list all admins
+- List all the relationships of a given user
+- Message displayed if mentorship relation request is accepted, rejected, terminated or deleted
```

Comment:
This seems unnecessary

Reply:
agree with this point @Jasmine9003 :) Could you remove this line please?